Selector/Input/index.jsx

```diff
@@ -0,0 +1,224 @@
+import React, { PropTypes } from 'react'
+import classNamesBind from 'classnames/bind'
+import compose from '../../lib/compose'
+import defaultStyles from './styles.scss'
+import * as fieldStates from '../../lib/features/fieldStates'
+import * as inlinedIcon from '../../lib/features/inlinedIcon'
+import * as stacking from '../../lib/features/stacking'
+import MouseflowExclude from '../../MouseflowExclude'
+import { Select } from '../../IconButton'
+import themeable from '../../decorators/themeable'
+import overridable from '../../decorators/overridable'
+
+const baseClass = 'input'
```

Comment:
Let's do a simple rename of this class to `selector-input`. Some people are not using CSS modules, (most everyone is using them in production, but it should work even if we are not using them)